app/controllers/api/v2/smart_class_parameters_controller.rb

```diff
@@ -36,6 +36,7 @@ def show
         param :override, :bool, :desc => N_("Whether the smart class parameter value is managed by Foreman")
         param :description, String, :desc => N_("Description of smart class")
         param :default_value, String, :desc => N_("Value to use when there is no match")
+        param :hidden_value, :bool, :desc => N_("Value to display when the parameter should be hidden")
```

Comment:
`hidden_value` is not the boolean it's "*****" this is how it's displayed for global parameters, I can add the boolean for both global parameters and here but I was trying to keep the same behavior.